target/linux/generic/config-3.18

```diff
@@ -3344,9 +3345,11 @@ CONFIG_SELECT_MEMORY_MODEL=y
 # CONFIG_SENSORS_HTU21 is not set
 # CONFIG_SENSORS_I5500 is not set
 # CONFIG_SENSORS_I5K_AMB is not set
+# CONFIG_SENSORS_IBM_CFFPS is not set
```

Comment:
Is there a local patch you added that adds the driver for the 3.18 kernel series? I can't find the symbol in the
3.18-stable tree

https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/drivers/hwmon/pmbus/Kconfig?h=linux-3.18.y

And according to [Linux Kernel Driver DataBase](https://cateee.net/lkddb/web-lkddb/SENSORS_IBM_CFFPS.html):

          found in Linux kernels: 4.14–4.20, 4.20+HEAD

I found a few more (not sure if they are all):  

        CONFIG_SENSORS_IR35221 (since 4.13)
        CONFIG_SENSORS_LTC3815 (since 4.5)
        CONFIG_SENSORS_MAX20751 (since 4.3)
        CONFIG_SENSORS_TPS53679 (since 4.14)

and there's
 
        CONFIG_SENSORS_LTC2978_REGULATOR

this one is hiding behind CONFIG_REGULATOR so unless the targets enables the REGULATOR it doesn't show up. (Also SENSORS_LTC2978_REGULATOR does not specify a default so it's likely generating a config prompt)